libnavui-dropin/src/test/java/com/mapbox/navigation/dropin/component/maneuver/ManeuverBehaviorTest.kt

```diff
@@ -18,9 +19,10 @@ class ManeuverBehaviorTest {
     @Test
     fun `when maneuver view height is updated`() {
         val sut = ManeuverBehavior()
+        val isVisible = Random.nextBoolean()
```

Comment:
> It's usually not a good idea to use random values in tests.

That's debatable. If the random value is present somewhere in the output in case the test fails, why not? Sure, you will have different inputs for different builds but that increases the probability of catching an error.
For booleans we can have 2 tests but what about ints and stuff?

Reply:
You should add such tests, so that every branch in your logic is covered. Testing any random values in between will not be necessary then, because every possible behavior is already tested. Anyway relying on random values to catch an error sounds weird, because it will likely be too late when a regression is finally detected.